src/features/hdr-tonemap.md

```diff
@@ -0,0 +1,85 @@
+# HDR, Tonemapping, Bloom, Dithering
+
+{{#include ../include/links.md}}
+
+HDR (High Dynamic Range) refers to the ability of the game engine to handle very
+bright lights or colors. Bevy supports HDR rendering. This means you can have
+objects with colors that go above `1.0`, very bright lights, or bright emissive
+materials. All of this is supported for both 3D and 2D.
+
+This is not to be confused with HDR display output, which is the ability to
+produce a HDR image to be displayed by a modern monitor or TV with HDR
+capabilities. Bevy has no support for this yet.
+
+## Camera HDR configuration
+
+There is a per-camera toggle that lets you decide whether you want Bevy to
+store the HDR data internally during rendering.
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/hdr.rs:hdr-config}}
+```
+
+If it is enabled, Bevy's intermediate textures will be in HDR format. This
+allows you to enable post-processing effects like Bloom, that make use of the
+HDR rendering data. Tonemapping will happen as a post-processing step.
+
+If it is disabled, the shaders are expected to do any [tonemapping][cb::tonemap]
+internally, and output standard RGB colors. The "physically based" materials can
+still handle bright lights and other "HDR" scenarios, but that information will
+not be stored, and effects like Bloom will not work.
+
+Make sure to also have [tonemapping][cb::tonemap] enabled, ideally with
+dithering.
+
+Bevy's HDR rendering is known to cause issues with MSAA. There might be visual
```

Comment:
What issues?

Reply:
I don't know. The API docs have a disclaimer, and so I thought the responsible thing would be to also pass on that information in Cheatbook. My goal here isn't to bring light to any specific issues, but rather to make people aware of the possibility that, should they notice anything wrong, turning off MSAA might be a reasonable thing to try.